examples/test-cra/src/storyshots.test.js

```diff
@@ -0,0 +1,5 @@
+import initStoryshots, { renderOnly } from 'storyshots';
+
+initStoryshots();
+
+// initStoryshots({ test: renderOnly });
```

Comment:
If this is an example of something that could be done for user's reference, maybe leave a comment? Otherwise, just remove it.